code/modules/projectiles/guns/projectile/pneumatic.dm

```diff
@@ -117,8 +117,12 @@
 
 	if(istype(user, /mob/living))
 		var/mob/living/M = user
-		if ((M_CLUMSY in M.mutations) && prob(50))
-			to_chat(M, "<span class='danger'>[src] blows up in your face.</span>")
+
+		if(M_HULK in M.mutations)
```

Comment:
![atom_2016-12-20_15-37-53](https://cloud.githubusercontent.com/assets/6526157/21363189/542063fe-c6ca-11e6-88a0-c0b0acfae411.png)
seems to be just another lame competing version of dexterity_check()?
Which excludes catbeasts, so that's nice